modules/ROOT/pages/configure-custom-serializers.adoc

```diff
@@ -152,31 +152,36 @@ Ensure that you are aware of the following limitations and considerations when u
 
 Serializers are not interoperable nor interchangeable. That means that if you decide to change the serializer your application uses, ensure that all messages in VM and JMS queues have been consumed and that those queues are empty by the time the new serializer takes over.
 
-The Kryo serializer won’t be able to read datagrams written by the Java serializer, and vice-versa. The same thing applies to persistent object stores; you cannot read with one serializer an entry generated with a different serializer.
+The Kryo serializer won't be able to read datagrams written by the Java serializer, and vice-versa. The same thing applies to persistent object stores; you cannot read with one serializer an entry generated with a different serializer.
 
 === Shared VM Connectors Use the Domain's Default Serializer
 
 Domains provide a way to share resources between applications. For example, you can define a VM connector on a domain to allow inter-app communication through VM message queues.
 
 However, serializers can be configured only at an application level; they cannot be configured at a domain level.
 
-If applications A and B communicate with each other through a VM connector defined on a domain to which both belong, but A serializes using Java and B using Kryo, the serialization works because that particular message is not serialized with the application’s serializer but the one the VM connector uses (the default serializer at domain level). This is good for a "plug-and-play experience," but you won’t be able to realize a performance improvement by having that shared VM connector use Kryo.
+If applications A and B communicate with each other through a VM connector defined on a domain to which both belong, but A serializes using Java and B using Kryo, the serialization works because that particular message is not serialized with the application's serializer but the one the VM connector uses (the default serializer at domain level). This is good for a "plug-and-play experience," but you won't be able to realize a performance improvement by having that shared VM connector use Kryo.
+
+=== Java 17 Support
+
+Starting with version 4.6, Mule runtime supports Java 17, which enforces JPMS. To achieve this capability, Mulesoft introduces several custom serializers that replace the default Kryo frameworks, because those default frameworks rely on mechanisms that break JPMS encapsulation.
```

Comment:
```suggestion
Starting with version 4.6, Mule runtime supports Java 17, which enforces JPMS. To achieve this capability, Mulesoft introduces several custom serializers that replace the default Kryo frameworks, because the default frameworks rely on mechanisms that break JPMS encapsulation.
```